flaskApi/file_db.py

```diff
@@ -0,0 +1,64 @@
+import json
+import os
+import file_system_db_util as util
+
+
+class Filedb:
+    def __init__(self):
+        # get dir to file folder (create if dont exist)
+        root_dir = os.path.dirname(os.getcwd())
+        self.UPLOAD_DIR = os.path.join(root_dir, 'files', 'uploads')
+        self.DOWNLOADS_DIR = os.path.join(root_dir, 'files', 'download')
+        os.makedirs(self.UPLOAD_DIR, exist_ok=True)
+        os.makedirs(self.DOWNLOADS_DIR, exist_ok=True)
+
+    def add(self, obj_to_save, name):
+        """
+          Add an object to be saved as a JSON file to 'uploads' folder
+
+          Args: obj_to_save (object): The object to be saved as a JSON file. name (str): The name used for generating
+          the filename.
+
+          Returns:
+              str: The unique ID associated with the saved file.(the finale name is 'unique_id_timestamp_original_name')
+          """
+        new_name, new_uid = util.generate_filename(name)
+        new_path = os.path.join(self.UPLOAD_DIR, new_name)
+
+        with open(new_path, 'w') as file:
+            json.dump({
+                'name': name,
+                'slides': obj_to_save
+            }, file)
+
+        return new_uid
+
+    def get_from_download(self, uid):
```

Comment:
Separate code to small functions.  Read more: https://softwareengineering.stackexchange.com/questions/308108/when-is-it-appropriate-to-make-a-separate-function-when-there-will-only-ever-be